src/Client.php

```diff
@@ -86,15 +84,14 @@ private function getExceptionFromResult($result)
     /**
      * Make HTTPS calls to Duo.
      *
-     * @param string  $endpoint   The endpoint we are trying to hit
-     * @param any     $request    Information to send to Duo
-     * @param boolean $user_agent (Optional)True if we want to send
```

Comment:
Was 'boolean' here inaccurate?

Reply:
Yes. `curl_setopt` with the `CURLOPT_USERAGENT` option takes a string (which sets the value of the "User-Agent: " header), not a boolean. See https://curl.se/libcurl/c/CURLOPT_USERAGENT.html and https://www.php.net/manual/en/function.curl-setopt.php.